workflows/prognostic_c48_run/runtime/factories.py

```diff
@@ -62,11 +64,14 @@ def get_tendency_prescriber(
     else:
         prescriber_config = config.tendency_prescriber
         tendency_variables = list(prescriber_config.variables.values())
-        if communicator.rank == 0:
```

Comment:
This was mainly to clean up from the last PR that the non-zero tile ranks don't need to all be opening the target dataset.